api/account/swagger-api_account.yaml

```diff
@@ -160,7 +160,22 @@ paths:
       responses:
         200:
           description: Successful operation.
-
+  /account/send-verify-email:
```

Comment:
Maybe we could remove the "send" in URL to have /account/verify-email as it would sounds more RESTful